### PR TITLE
checksum 0.1.0

### PR DESCRIPTION
**Pull request by denkisikinoneko**
Created via `alr publish` with `alr 2.0.1`

### CODE REVIEW

File: index/ch/checksum/checksum-0.1.0.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name = "checksum"
+description = "Algorithms for error-checking"
+version = "0.1.0"
+
+authors = ["Miko Elbrecht"]
+maintainers = ["Miko Elbrecht <tcgytsc@gmail.com>"]
+maintainers-logins = ["ATPStorages"]
+licenses = "GPL-3.0-or-later"
+website = "https://javart.zip"
```
Comment (mosteo) [inline index/ch/checksum/checksum-0.1.0.toml]: I get site unreachable?

Comment (denkisikinoneko) [reply index/ch/checksum/checksum-0.1.0.toml]: Site I own that I plan to set up a web server on in the future. I can add a DNS record for validity, if you'd like

Comment (mosteo) [reply index/ch/checksum/checksum-0.1.0.toml]: If it isn't going to be working imminently, I'd leave the website pointing to the repository for the time being. Once the one at https://javart.zip is up and running, updating this bit of metadata won't require a new release.

Comment (denkisikinoneko) [reply index/ch/checksum/checksum-0.1.0.toml]: Updated to git repository. Didn't know that about the metadata, thank you

**Review by mosteo — APPROVED** [review]

**Pull request merged**